src/compiler.js

```diff
@@ -64,11 +64,10 @@ if (settingsFile) {
   }
 }
 
-EXPORTED_FUNCTIONS = set(EXPORTED_FUNCTIONS);
-EXCEPTION_CATCHING_ALLOWED = set(EXCEPTION_CATCHING_ALLOWED);
```

Comment:
Is this not necessary anymore?

Reply:
Yes, it seems the last usage of this set from JS library code was removed back in 95e718e483089e4c9df46486dbc4f7cccc12d648 (when it was called EXCEPTION_CATCHING_WHITELIST).